src/core/test/system/system.t.sol

```diff
@@ -321,4 +324,23 @@ contract SystemTest is TestUtils, DSTest {
         borrower.doBorrow(loan, 100);
         assertEq(currency.balanceOf(borrower_), 100);
     }
+
+    function testSupply() public {
+        supply(100 ether, 10 ether);
```

Comment:
I think we should use variable and not constant numbers if we have more than one parameter.

```javascript
uint investorBalance;
uint supplyAmount;
supply(investorBalance, supplyAmount);
```